src/test/java/com/redhat/cloud/notifications/OpenApi2Test.java

```diff
@@ -0,0 +1,68 @@
+package com.redhat.cloud.notifications;
+
+import com.reprezen.kaizen.oasparser.OpenApi3Parser;
+import com.reprezen.kaizen.oasparser.model3.OpenApi3;
+import com.reprezen.kaizen.oasparser.val.ValidationResults;
+import io.quarkus.test.common.http.TestHTTPResource;
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.Assert;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import java.net.URL;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.RestAssured.when;
+
+/**
+ * Do some basic verification of the created
+ * openapi.*.json files
+ */
+@QuarkusTest
+@Tag("integration")
+public class OpenApi2Test {
+
+    // QuarkusTest will inject the host+port for us.
+    @TestHTTPResource(TestConstants.API_NOTIFICATIONS_V_1_0 + "/openapi.json")
+    URL nUrl;
+
+    @TestHTTPResource(TestConstants.API_NOTIFICATIONS_V_1 + "/openapi.json")
+    URL nUrl1;
+
+    @TestHTTPResource(TestConstants.API_INTEGRATIONS_V_1_0 + "/openapi.json")
+    URL iUrl;
+
+    @TestHTTPResource(TestConstants.API_INTEGRATIONS_V_1 + "/openapi.json")
+    URL iUrl1;
+
+    @TestHTTPResource("/api/doesNotExist/v1.0/openapi.json")
+    URL badUrl;
+
+    @Test
+    public void validateOpenApi() throws Exception {
+
+        URL[] urls = {nUrl, iUrl, nUrl1, iUrl1};
+
+        for (URL url : urls) {
+            System.out.printf("Validating OpenAPI Model at %s\n", url);
```

Comment:
Are these System prints necessary? They pollute every test run. Should they be enabled only if investigating the problem with test "validateOpenApi" failing?